xhr.bs

```diff
@@ -930,13 +933,27 @@ return <a>this</a>'s <a>cross-origin credentials</a>.
    is not <a>allowed to use</a> the "<code><a>sync-xhr</a></code>" feature, then run
    <a>handle response end-of-body</a> for <a>this</a> and a <a>network error</a>, and then return.
 
+   <li><p>Let <var>response</var> be a <a for=/>network error</a>.
+
+   <li><p>Let <var>doesNotBlockTask</var> be false.
+
    <li>
-    <p>Let <var>response</var> be the result of
-    <a for=/>fetching</a> <var>req</var>.
+    <p>Let <var>processResponse</var>, given a <var>fetchResponse</var>, be these steps:
+
+    <ol>
+     <li><p>Set <var>response</var> to <var>fetchResponse</var>.
+
+     <li><p>Set <var>doesNotBlockTask</var> to true.
+    </ol>
+
+   <li><p><a for=/>Fetch</a> <var>req</var> with <a for=fetch><i>processResponse</i></a>
+   set to <var>processResponse</var> and <a for=fetch><i>useParallelQueue</i></a> set to true.
+
+   <li><p>Wait until either <var>doesNotBlockTask</var> is true or <a>this</a>'s <a>timeout</a> is
+   not 0 and <a>this</a>'s <a>timeout</a> milliseconds have passed since now.
```

Comment:
I'd prefer saving "now" in a previous step. Although the intention is clear, as written one could interpret this as never happening.